{
  "Header": {
    "Original": "CVE-2020-25828 (An issue was discovered in MediaWiki before 1.31.10 and 1.32.x through ...)",
    "Line": 149833,
    "ID": "CVE-2020-25828",
    "Description": "(An issue was discovered in MediaWiki before 1.31.10 and 1.32.x through ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-4767-1 DLA-2379-1}",
      "Line": 149834,
      "Type": "xref",
      "Bugs": [
        "DSA-4767-1",
        "DLA-2379-1"
      ]
    },
    {
      "Original": "- mediawiki 1:1.35.0-1",
      "Line": 149835,
      "Type": "package",
      "Package": "mediawiki",
      "Kind": "fixed",
      "Version": "1:1.35.0-1"
    },
    {
      "Original": "NOTE: https://lists.wikimedia.org/pipermail/wikitech-l/2020-September/093888.html",
      "Line": 149836,
      "Type": "NOTE",
      "Description": "https://lists.wikimedia.org/pipermail/wikitech-l/2020-September/093888.html"
    },
    {
      "Original": "NOTE: https://phabricator.wikimedia.org/T115888",
      "Line": 149837,
      "Type": "NOTE",
      "Description": "https://phabricator.wikimedia.org/T115888"
    }
  ]
}